### PR TITLE
Ibm getting started

### PR DESCRIPTION
**Pull request by devpramod**

### CODE REVIEW

File: getting-started/README.md [hunk 1]
```diff
@@ -1,110 +1,108 @@
 # Getting Started with OPEA
+This document specifically details the steps for deploying services on IBM Cloud, providing a tailored guide to help you leverage IBM's cloud infrastructure to deploy the ChatQnA application from OPEA GenAI Examples. For additional deployment targets, see the [ChatQnA Sample Guide](https://opea-project.github.io/latest/examples/ChatQnA/ChatQnA_Guide.html)
+
+## Understanding OPEA's Core Components
+
+Before moving forward, it's important to familiarize yourself with two key elements of OPEA: GenAIComps and GenAIExamples.
+
+- GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (language learning models), embedding, and reranking, among others.
```
Comment (mkbhanda) [inline getting-started/README.md:8]:
```suggestion
- GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (large language models), embedding, and reranking, among others.

```

File: getting-started/README.md [hunk 2]
```diff
@@ -1,110 +1,108 @@
...
-## Understanding OPEA's Core Components
+3. Under Stock Images, select Ubuntu 24.04 (`ibm-ubuntu-24-04-6-minimal-amd64-1`)
 
-Before moving forward, it's important to familiarize yourself with two key elements of OPEA: GenAIComps and GenAIExamples.
+4. Select a virtual server.
+> **Note:** We recommend selecting a 3-series instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor, such as `bx3d-16x80` or above. For more information on virtual servers on IBM cloud visit [Intel® solutions on IBM Cloud®](https://www.ibm.com/cloud/intel).
 
-- GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (language learning models), embedding, and reranking, among others.
-- While GenAIComps offers a range of microservices, GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications.
+5. Create or add an ssh key for the instance.
```
Comment (mkbhanda) [inline getting-started/README.md:25]:
```suggestion
5. Add an SSH key to the instance, if necessary, create one first.
```

File: getting-started/README.md [hunk 3]
```diff
@@ -1,110 +1,108 @@
...
 
-## Visual Guide to Deployment
-To illustrate, here's a simplified visual guide on deploying a ChatQnA GenAIExample, showcasing how you can set up this solution in just a few steps.
+6. Click on `Create virtual server` at the bottom right.
```
Comment (mkbhanda) [inline getting-started/README.md:27]: Do you want to stay agnostics of UI changes, if yes, leave out the "at bottom right"

File: getting-started/README.md (hunk 2 above)
Comment (mkbhanda) [reply getting-started/README.md:25]: note the capitalization of SSH

File: getting-started/README.md (hunk 1 above)
Comment (mkbhanda) [reply getting-started/README.md:8]: See for LLM expansion https://en.wikipedia.org/wiki/Large_language_model

File: getting-started/README.md [hunk 4]
```diff
@@ -1,110 +1,108 @@
...
-## Setup ChatQnA Parameters
-To deploy ChatQnA services, follow these steps:
+8. `ssh` into the instance using the floating IP (`ssh -i <key> ubuntu@<floating-ip>`)
 
+
+
+## Deploy the ChatQnA Solution
+Use the command below to install docker on a clean virtual machine
+```bash
+wget https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh
```
Comment (mkbhanda) [inline getting-started/README.md:38]: This script might move ..
Why not a put a copy click thing on page and reproduce the instructions in the install_docker.sh inline?

File: getting-started/README.md [hunk 5]
```diff
@@ -1,110 +1,108 @@
...
-# on Gaudi
-source ./docker_compose/intel/hpu/gaudi/set_env.sh
-# on Nvidia GPU
-source ./docker_compose/nvidia/gpu/set_env.sh
+Now we can start the services
+```bash
+docker compose up -d
 ```
+It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running.
+For example to check the logs for the `tgi-service`:
```
Comment (mkbhanda) [inline getting-started/README.md:66]: Check if all the services are up and running. 
> docker ps -a
Expect the following container images:

image

File: getting-started/README.md [hunk 6]
```diff
@@ -1,110 +1,108 @@
...
-- docker pull opea/chatqna:latest
-- docker pull opea/chatqna-ui:latest
+### Interact with ChatQnA
 
-In the following cases, you will need to build the docker image from source by yourself.
+You can interact with ChatQnA via a browser interface:
+* Under `Infrastructure` in the left pane, go to `Network/Security groups/<Your Security Group>/Rules`
+* Select `Create`
+* Enable inbound traffic for port 80
+* Open a browser and navigate `http://{external_public_ip}:80` to view the ChatQnA interface.
```
Comment (tomlenth) [inline getting-started/README.md:82]: recommend making this more explicit by modifying this step to be: 

* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your externally facing IP address in the following:  `http://{external_public_ip}:80'.

**Review by tomlenth — CHANGES_REQUESTED** [review]
minor change to the interface commands.

File: getting-started/README.md [hunk 7]
```diff
@@ -1,110 +1,108 @@
...
+```bash
 curl http://${host_ip}:8888/v1/chatqna \
     -H "Content-Type: application/json" \
     -d '{
         "messages": "What is the revenue of Nike in 2023?"
     }'
 ```
-This command will provide the response as a stream of text. You can modify the `message` parameter in the `curl` command and interact with the ChatQnA service.
+ChatQnA provides the answer to your query as a text stream.
+
```
Comment (mkbhanda) [inline getting-started/README.md:104]: Do add a step to upload content to RAG system

Use the curl command below to upload a file.

curl -X POST "http://${host_ip}:6007/v1/dataprep" \
     -H "Content-Type: multipart/form-data" \
     -F "files=@./nke-10k-2023.pdf"

**Review by mkbhanda — CHANGES_REQUESTED** [review]

File: getting-started/README.md (hunk 4 above)
Comment (devpramod) [reply getting-started/README.md:38]: @mkbhanda will add a fair few bit of lines - https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh

File: getting-started/README.md (hunk 7 above)
Comment (devpramod) [reply getting-started/README.md:104]: @mkbhanda Can we add this in the next iteration? I'd have to validate it on IBM cloud

File: getting-started/README.md (hunk 1 above)
Comment (devpramod) [reply getting-started/README.md:8]: copied this over from the previous GS! changed it

**Review by tomlenth — APPROVED** [review]
thanks for the hard work!

File: getting-started/README.md (hunk 4 above)
Comment (mkbhanda) [reply getting-started/README.md:38]: Actually no, you have 3 lines here, and downloading docker stuff directly is 1 more line.

File: getting-started/README.md (hunk 7 above)
Comment (mkbhanda) [reply getting-started/README.md:121]: sure

**Review by mkbhanda — APPROVED** [review]
LGTM

**Pull request merged**